utils/RunONNXModel.py

```diff
@@ -158,6 +160,14 @@ def check_non_negative(argname, value):
     help="Path to a folder containing reference inputs and outputs stored in protobuf."
     " If --verify=ref, inputs and outputs are reference data for verification",
 )
+data_group.add_argument(
+    "--load-ref-from-numpy",
+    metavar="PATH",
+    type=str,
+    help="Path to a python script that defines variables inputs and outputs that are a list of numpy arrays. "
+    " For example, inputs = [np.array([1], dtype=np.int64), np.array([2], dtype=np.float32]."
+    " Variable outputs can be omitted if --verify is not used.",
```

Comment:
>Is outputs optional?
Please add a bit of details.

@AlexandreEichenberger is it not enough and confusing?